pages/index.tsx

```diff
@@ -37,20 +37,20 @@ const Index: React.FC<IndexPageProps> = (props: IndexPageProps) => {
               <h1 className="heading-1">Aotearoa</h1>
             </div>
             <RegionDropdown
-              selectedRegion={region}
-              onChange={(regionId) => selectedRegion(regionId)}
+              selectedRegion={query.dhbs}
+              onChange={(id) => setQuery({ dhbs: id })}
             />
           </PageContainer>
         </section>
         <section>
           <PageContainer>
             <DhbsVaccineDoseDataList
-              key={region}
+              key={query.dhbs}
               dhbsVaccineDoseData={allDhbsVaccineDoseData.filter((dhb) => {
-                if (region === "all") {
+                if (query.dhbs === "all") {
                   return true;
                 }
-                return dhb.regionIds.includes(region);
+                return dhb.regionIds.includes(query.dhbs as DhbRegionId);
```

Comment:
Need to cast since dhbs can be a string - should be fine since `.includes` won't fail 🙏